modules/imaging_browser/php/imaging_session_controlpanel.class.inc

```diff
@@ -105,13 +105,20 @@ class Imaging_Session_ControlPanel
             $params['PCandID'] = $timePoint->getCandID();
             $params['PVL']     = $timePoint->getVisitLabel();
         }
-        $tarchiveIDs = $DB->pselect(
-            "SELECT TarchiveID
+        // To enable DICOM download: Add pair: tarchiveID, ArchiveLocation
+        $qresult = $DB->pselect(
+            "SELECT TarchiveID, ArchiveLocation 
             FROM tarchive 
             WHERE PatientName LIKE $ID",
             $params
         );
-        $subjectData['tarchiveids'] = $tarchiveIDs;
+
+        $tarIDToTarLoc =array();
+        foreach ($qresult as $v) {
+            $tarchiveID = $v['TarchiveID'];
+            $tarIDToTarLoc[$tarchiveID] =$v['ArchiveLocation'];
```

Comment:
I'm surprised this passed PHPCS

Reply:
I don't really know;
I added an extra space after the `=` sign, so it is a manual change; and this passes phpcs if this was your main concern (I still do not know how it passed before)
not sure if the other issue is also the line before where the `==` is not aligned with the line underneath it, but when I did force it to manually be aligned with the line below, it actually gave me phpcs error!